content/en/profiler/enabling/supported_versions.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Language and Tracer Versions for Profiler Features
+kind: documentation
+disable_sidebar: true
+further_reading:
+- link: "/profiler/enabling"
+  tag: "Documentation"
+  text: "Enabling Profiler"
+---
+
+The following table shows a summary of the features available for each language runtime. 
+- **Minimum versions** are the minimum required to get access to a particular feature-If you have an earlier version, that feature is not supported. 
+- **Recommended versions** are the runtime and tracer versions that give you access to **all** the supported features. Generally speaking, it's best if you update to the latest version of all tracers.
+
+More details about runtime support and profile types is available on each language's profiler setup page (click the language heading in the table to access).
+
+|                                   | [Java][1]  | [Python][2]  |  [Go][3]   |  [Ruby][4] |   [Node.js][5]  |  [.NET][6]   |   [PHP][7]  | [Rust/C/C++][8] |
+|-----------------------------------|:-------:|:-------:|:------------:|:------:|:---------:|:-------:|:------:|:----------:|
+| Operating systems                 | | | | Linux only | | | CentOS&nbsp;7+ | Linux&nbsp;v4.17+ |
```

Comment:
PHP supports other Linux versions as well, I do think this is not the right way of representing this information

Reply:
+1 on this being incomplete/over-simplified. Missing things like supported architectures and GNU libc vs MUSL libc issues for profilers with native code that will require compilation.

Reply:
Thanks for the feedback. I went ahead and removed the OS row for now.